README.rst

```diff
@@ -592,6 +606,10 @@ The following flavors of Linux are supported
 Changes
 -------
 
+2.2.5
+
+- Correct processing of iproute interfaces. (ZPS-1695)
```

Comment:
Can you be more specific about what was incorrect before? We want the user to know if they're experiencing the problem that was fixed.